components/global/global.js

```diff
@@ -1,44 +1,63 @@
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import {
-  unstable_renderSubtreeIntoContainer, // eslint-disable-line
-  unmountComponentAtNode,
+  createPortal,
 } from 'react-dom';
 
 /**
  * Global component
  * React component for transportation of modals, lightboxes, loading bars... to document.body
  */
 class Global extends Component {
-  componentDidMount() {
-    if (this.props.noscroll) {
-      global.window.document.body.classList.add('ui-global_noscroll');
-    }
+  constructor(props) {
+    super(props);
 
+    this.isSettedNoScroll = false;
     this.target = global.window.document.createElement('div');
     this.target.classList.add('ui-global');
     global.window.document.body.appendChild(this.target);
-    this.componentDidUpdate();
   }
 
-  componentDidUpdate() {
-    unstable_renderSubtreeIntoContainer(this, (
-      <div className={this.props.className}>
-        {this.props.children}
-      </div>
-    ), this.target);
+  toggleGlobalNoscroll(flag) {
+    const body = global.window.document.body;
+    if (!this.isSettedNoScroll) {
+      return;
+    }
+    flag
+      ? body.classList.add('ui-global_noscroll')
```

Comment:
is it possible to have 2 `Globals` at the same time? like opened modal from the sliding panel? How will it work then?

Reply:
I added an example https://github.com/Travix-International/travix-ui-kit/pull/226/commits/c68674c0048d67a6ac59720ea951a1eb7d8ea480